src/main/java/com/palantir/gradle/revapi/config/GroupNameVersion.java

```diff
@@ -37,17 +36,17 @@ default String asString() {
 
     @JsonCreator
     static GroupNameVersion fromString(String groupNameVersionString) {
-        List<String> split = Splitter.on(':').splitToList(groupNameVersionString);
+        String[] split = groupNameVersionString.split(":");
```

Comment:
this has a bug where `::` will be considered as a single separator, I don't really want to go back to a world where we have to think about these bugs

Reply:
Alternative here could be to just use a regex to require the (group):(name):(version) thing we expect?